test/media/adaptation_set_criteria_unit.js

```diff
@@ -19,7 +19,12 @@ describe('AdaptationSetCriteria', () => {
         });
       });
 
-      const builder = new shaka.media.PreferenceBasedCriteria('en', '', 0, '');
+      const builder = new shaka.media.PreferenceBasedCriteria(
+          /* language= */ 'en',
```

Comment:
Thanks for adding these comments to the test. The arguments list for this function was already starting to get unwieldly.